### DISCUSSION TITLE
[Bug report] Error Syncing site with Netlify

### DISCUSSION THREAD

**Discussion by LandAgency** [GetPublii/Publii#1488] [Bug Report]
### Operating system

Windows 11

### Publii version

v.0.45.2 (build 16609)

### Issue type

Application

### Bug description

Keep getting an error when Syncing site to Netlify. Deployment process log:

[Sat, 20 Apr 2024 00:38:26 GMT] Netlify ERROR: TypeError: Cannot read properties of undefined (reading 'id')

**Comment by zomgs** [comment]
I ran into this multiple times as well ever since updating to the latest; sometimes it was resolved by making a new token in "Netlify", but not a 100% of the time.

**Comment by dziudek** [comment]
Hi,

Please go to your server settings and please check if the Site ID is provided. On the netlify sync code `.id` refers only to the Site ID option

**Reply by LandAgency** [reply]
It started working for a second but now it's back to the same error. I have the Netlify Site ID copied into the Publii Site ID field. I'm on the latest Publii build. I tried a new token but that didn't work.
